toxcore/query.c

```diff
@@ -0,0 +1,190 @@
+/** query.c
+ *
+ * Makes requests for information using the DHT or Onion as approitate.
+ *
+ * This file should include a licence, but if you're reading this, it doesn't yet so everything in
+ * this file is donated to both tox, toktok, and you under a BSD or MIT licence at any option.
```

Comment:
@zetok GrayHatter can do whatever he want's with his code, IMHO. I don't see why this couldn't works like that